docs/lib/graphqlapi/fragments/flutter/authz/20_oidc.md

```diff
@@ -0,0 +1,5 @@
+<amplify-callout>
+
+OIDC mode is not yet supported for Flutter. We are actively working on this. 
```

Comment:
If possible, can you include a link to an open GitHub issue, so that people can track the feature?

Reply:
offlien talk, will coordinate another RFC with Ashish